cdap-docs/application-templates/source/etl/templates/sinks/batch/cube.rst

```diff
@@ -16,7 +18,9 @@ sink.
 
 .. rubric:: Use Case
 
-TODO: Fill me out
+A BatchCubeSink is used to write data into a Cube from batch-enabled data sources to allow further data analysis.
+E.g. with BatchCubeSink you can periodically upload data from PartitionedFileSet into a Cube to perform complex
```

Comment:
For example, you can periodically upload data from a PartitionedFileSet into a Cube to perform complex